browser/ui/webui/settings/brave_import_data_handler.h

```diff
@@ -37,11 +39,15 @@ class BraveImportDataHandler : public ImportDataHandler,
   BraveImportDataHandler& operator=(const BraveImportDataHandler&) = delete;
 
  private:
-#if BUILDFLAG(IS_MAC)
   // ImportDataHandler overrides:
   void StartImport(const importer::SourceProfile& source_profile,
                    uint16_t imported_items) override;
-
+  void StartImportImpl(const importer::SourceProfile& source_profile,
```

Comment:
nit: please add blank line between overriden method part and the others.

Reply:
done